custom_components/plugwise/__init__.py

```diff
@@ -1,30 +1,35 @@
 """Plugwise platform for Home Assistant Core."""
+
 from __future__ import annotations
 
 from typing import Any
 
 from plugwise.exceptions import PlugwiseError
-import voluptuous as vol
+import voluptuous as vol  # pw-beta delete_notification
 
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import Platform
-from homeassistant.core import HomeAssistant, ServiceCall, callback
+from homeassistant.core import (
+    HomeAssistant,
+    ServiceCall,  # pw-beta delete_notification
+    callback,
+)
 from homeassistant.helpers import device_registry as dr, entity_registry as er
 
 from .const import (
     CONF_REFRESH_INTERVAL,  # pw-beta options
     DOMAIN,
     LOGGER,
     PLATFORMS,
-    SERVICE_DELETE,
+    SERVICE_DELETE,  # pw-beta delete_notifications
 )
 from .coordinator import PlugwiseDataUpdateCoordinator
 
 type PlugwiseConfigEntry = ConfigEntry[PlugwiseDataUpdateCoordinator]
 
 
 async def async_setup_entry(hass: HomeAssistant, entry: PlugwiseConfigEntry) -> bool:
-    """Set up the Plugwise Device from a config entry."""
+    """Set up Plugwise components from a config entry."""
```

Comment:
Should we write here: `"""Set up the Plugwise Integration from...`?

Reply:
It's a complete mixed bag ... I've rewritten most of them to `Set up Plugwise x from config entry` now though with your next comment we might need more of those. Core is a mix of Plugwise and Smile and ConfigEntry vs config entry :)
Maybje just `Setup Plugwise from a config Entry`?

Reply:
Maybe `Set up the Plugwise Integration from a config entry`?
Good to add `config entry` :+1: